packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -82,8 +82,8 @@ watch(
 			}
 			configs = await getModelConfigurationsForModel(modelId);
 		}
-		// Auto append output
-		if (configs[0]?.id) {
+		// Auto append output if and only if there isnt already an output
+		if (!props.node.outputs.at(0)?.value && configs[0]?.id) {
```

Comment:
I _think_ this may cause issues with this flow:
1. We have 2 models `Model_A`, `Model_B`
2. Attach `Model_A` to a model config node
3. Unlink `Model_A` from model config node
4. Attach `Model_B` to the same model config node

Will opening the model config here show the default configuration and output from `Model_B`? It does that currently

Reply:
Good question, we should clarify what is the expected behaviour here!

Reply:
I have now updated this to be more in line with how other operators work:
```
We have 2 models Model_A, Model_B
Attach Model_A to a model config node
Unlink Model_A from model config node
Attach Model_B to the same model config node
Currently we still have model_a's configuration.
Now make any changes and click save for a new output 
Now we will have model_b's configuration.
```

This is similar to if we 
```
plug in model_config_a to simulate run
back out
unplug model_config_a
plug in new model config_b
now simulate run's output does not line up with the model config thats technically connected

```

Reply:
I think I raised this awhile ago and this was Dan's solution but I will reach out to design channel

Reply:
Based off of the chat in the design channel it seems like having this working like the other operators is okay as we may want to overhaul how they work as a whole.
Please let me know if you disagree

Reply:
Sure, as long as everyone's on the same page seems good to me